jetstream/statistics.py

```diff
@@ -112,7 +117,11 @@ def name(cls):
         return re.sub("([a-z0-9])([A-Z])", r"\1_\2", name).lower()
 
     def apply(
-        self, df: DataFrame, metric: str, reference_branch: Optional[str]
+        self,
+        df: DataFrame,
+        metric: str,
+        reference_branch: Optional[str],
+        experiment: "config.ExperimentConfiguration",
```

Comment:
Just passing in the experiment object seems to be the easiest way for now

Reply:
This could replace the `reference_branch` argument as well.